### ISSUE TITLE
When you send prep through salesforce - profile and about the job does not get included automatically 

### ISSUE THREAD

**Issue by matanaim** [onlylions/suggestions#15]
I always have to go back into the jobs page and copy and paste it in. Would be good if it included it automatically.

**Comment by b3ntly** [comment]
Yup and issues #1 , #7 , #9 all represent this same problem. The sync between our API and Salesforce is broken in such a way that wasn't obvious during development. It'll work for the first ~3 hours after a server restart and then get out of sync.

I'm writing a fix for it that should be out in the next few days™️ then any changes that happen in Salesforce should be shown instantly on the leads, intros, and jobs pages, instantly.

**Comment by matanaim** [comment]
Ok thanks Ben!! I guess its also good to see that a couple people report it
- shows the level of urgency
